Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/TypeBindingGenerator.cpp

```diff
@@ -225,17 +213,34 @@ void GenerateBlueprintStructEnumDefinition(FCodeWriter& Writer, UProperty* Prope
 			return;
 		}
 
-		GeneratedTypes.Add(Property->GetCPPType());
+		VisitedTypes.Add(Property->GetCPPType());
 
-		Writer.PrintNewLine();
-		Writer.Printf("enum %s", *Enum->GetName());
-		Writer.Printf("{").Indent();
+		WriterCallback(Property, Enum);
+	}
+}
+
+void VisitAllBlueprintDefinedStructures(const FUnrealRPCsByType& RPCsByType, const FUnrealFlatRepData& RepData, BPStructTypesAndPaths& VisitedStructInfo, TFunction<void(const UProperty*, const UField*)> WriterCallback)
+{
+	// Visit all replicated properties
+	TSet<FString> VisitedBPTypes;
+	for (EReplicatedPropertyGroup Group : GetAllReplicatedPropertyGroups())
+	{
+		for (auto& RepProp : RepData[Group])
```

Comment:
It's really difficult to tell if these `auto`s (here and below) are correct, or should be `const`, or by value.

Reply:
Yep good point and this code doesn't adhere with our coding standard. Will retro fix this file.